packages/manager/modules/catalog-price/package.json

```
@@ -15,10 +15,11 @@
     "lodash": "^4.17.15"
   },
   "peerDependencies": {
+    "@ovh-ux/manager-config": "^0.4.0",
```

```suggestion
    "@ovh-ux/manager-config": "^0.4.0 || ^0.50",
```
or
```suggestion
    "@ovh-ux/manager-config": "^0.4.0 || ^1.0.0",
```
depending on the commit type applied to the package

---

We should release theses packages in `1.x.x`...

---

Done, rebased on `develop`